web/static/css/components/_Achievement.scss

```diff
@@ -0,0 +1,20 @@
+.Achievement {
+  margin-bottom: $vertical-spacing;
+  text-align: left;
+  display: flex;
+  align-items: center;
+
+  &-icon {
+    font-size: 3em;
```

Comment:
Properties should be ordered position, font-size, opacity

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)